### ISSUE TITLE
[Suggestion] Option to not mute during gameplay

### ISSUE THREAD

**Issue by blueYOSHI9000** [tmarenko/GenshinImpact_PaimonShutUp#10]
Sometimes characters talk during regular dialogue and sometimes on the field during actual gameplay. Not having any sound during gameplay (especially when fighting) can be a bit distracting, and usually there's enough sounds already so Paimons less of an issue there.
One solution I can think of would be to add an option to only mute the game when the "Auto" button is displayed in the top left corner.

**Comment by tmarenko** [comment]
Muting Paimon during gameplay was deliberate decision because sometimes she speaks out of combat (usually during main quests). But yeah I can see that this could be quite distracting. 
I'm not sure how to implement this "option" into existing code. One way would be to have another configuration parameter in `settings.cfg` but I don't really like it.
Probably will put two versions into next release and let you pick what to use.

For now here is branch: https://github.com/tmarenko/GenshinImpact_PaimonShutUp/tree/mute-dialogue-only
and here is build: [GenshinImpact_PaimonShutUp-mute-dialogue-only-build.zip](https://github.com/tmarenko/GenshinImpact_PaimonShutUp/files/10953345/GenshinImpact_PaimonShutUp-mute-dialogue-only-build.zip)

**Comment by blueYOSHI9000** [comment]
Thank you for the quick answer and fix! Been playing with it and it works perfectly!

As for the config, if the problem is it possibly being too confusing, maybe something like this would work?
Putting a bit of space between the previous option and also labeling it in a easier way to understand (at least I'd assume "Mute Overworld Dialogue" is easier to understand than "Only Mute Dialogue", this also means the boolean would be inversed though).
```
[config]
# available: chi_sim, chi_tra, eng, kor, jpn, spa, fra, rus, tha, vie, deu, ind, por
language=eng

#whether Paimon dialogue should also be muted when talking on the overworld without the dialogue box, can be true or false
mute_overworld=true

[genshin_window_name]
...
```

**Comment by tmarenko** [comment]
Added in https://github.com/tmarenko/GenshinImpact_PaimonShutUp/commit/410635f455c9b98d1469c544e1680ab0c579acce
Released in [1.5.0](https://github.com/tmarenko/GenshinImpact_PaimonShutUp/releases/tag/1.5.0)
